src/core/global_include.cpp

```diff
@@ -158,4 +164,18 @@ bool are_equal(double one, double two)
     return (std::fabs(one - two) < std::numeric_limits<double>::epsilon());
 }
 
+FCUTime::FCUTime() {}
```

Comment:
What's `FCU`? Can we have a name that's easier to understand? Maybe `Autopilot`?

Reply:
That's make sense :)